BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.h

```diff
@@ -5,20 +5,29 @@
 
 #import "BOXRequestWithSharedLinkHeader.h"
 
+typedef NS_OPTIONS(NSUInteger, BOXRepresentationRequestOptions) {
+    BOXRepresentationRequestNoOptions                          = 0,       // Default option if no options set or permission to file not granted
+    BOXRepresentationRequestOriginal                           = 1 << 0,  // Request original content url with file information request
```

Comment:
I think if we use `///` comments on the line above, I think that will get us short descriptions in the Xcode autocomplete box. Might be nice.

Reply:
done